### PR TITLE
Added the collectables and not collectables.

### PR DESCRIPTION
**Pull request by NeplayGames**
This branch consist of the following updates

1. Created two different kinds of collectables with boolean.
2. Created prefabs for apple and rotten apple.

### CODE REVIEW

File: Assets/BasketController.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class BasketController : MonoBehaviour
+{
+    void OnCollisionEnter(Collision collision){
+        if(collision.gameObject.TryGetComponent<ICollectables>(out ICollectables collectables))
+        {
+            if(collectables.IsCollectable())
+                Destroy(collision.gameObject);
+            else{
+                print("You lose");
```
Comment (NeplayGames) [inline Assets/BasketController.cs:14]: - [ ] Dont add print

Comment (Chaospeedy) [inline Assets/BasketController.cs:14]: - [ ] Remove this line

Comment (NeplayGames) [reply Assets/BasketController.cs:14]: Done

**Review by NeplayGames — COMMENTED** [review]
Made thoses changes

**Review by Chaospeedy — APPROVED** [review]

**Pull request merged**